Source/com/drew/metadata/mp4/Mp4HandlerFactory.java

```diff
@@ -51,6 +52,8 @@ public Mp4HandlerFactory(Mp4Handler<?> caller)
             return new Mp4HintHandler(metadata, context);
         } else if (type.equals(HANDLER_TEXT_MEDIA)) {
             return new Mp4TextHandler(metadata, context);
+        } else if (type.equals(HANDLER_NRT_META_MEDIA)) {
+            return new Mp4NRTMetaHandler(metadata, context);
```

Comment:
I'm not sure if this is the best naming of the old "meta" handler. This is what ExifTool calls it [NRT Metadata], but I think we should verify if this naming is proper.

https://exiftool.org/TagNames/QuickTime.html